src/App.css

```diff
@@ -13,11 +13,15 @@ a:visited {
 
 main {
   text-align: center;
-  position: relative;
 }
 
 nav {
+  width: 100%;
+  height: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: flex-end;
```

Comment:
Nice usage of flex here 🙌🏽  Little note: Since `row` is the default `flex-direction` it could be omitted.